Add new notebook to run LVV-T2552 - Rotator Duty Cycle

b1quint opened the pull request:
This PR contains the notebook that will run the Rotator Duty Cycle Test. We need to review if the targets az/el correspond correctly to the desired tracking speed. We also need to check that the cells reflect the steps precisely.

b1quint committed:
Add new notebook to run LVV-T2552 - Rotator Duty Cycle

For now, this is just a draft of the notebook that wil run the
Rotator Duty Cycle Test. We need to review if the targets az/el
correspond correctly to the desired tracking speed. We also need
to review that the cells reflect the steps correctly.

hdrass commented:
Hi Bruno! Looks good to me! This seems to be are a very complete test.

tribeiro reviewed (changes requested):
This is a great first step! thank you so much @b1quint... I made a few comments in nbreview (https://app.reviewnb.com/lsst-ts/ts_notebooks/blob/tickets/DM-34084%2Fprocedures%2FLVV-T2572%20-%20Rotator%20Duty%20Cycle%20Tests.ipynb%2F/file), please take a look at them...

b1quint committed:
Update notebook to run the Rotator Duty Cycle Test

This update follows the request for changes:
 - Remove the stop-tracking commands after each slew.
 - Use mtcs.move_rotator
 - Use RA/Dec for each slew with slew_icrs
 - Fix not-showing image
 - Minor other reformat to help reading.

b1quint commented:
I pushed a new commit with the requested changes. Please, have a look and let me know if I should apply any further changes.

tribeiro commented on procedures/LVV-T2572 - Rotator Duty Cycle Tests.ipynb:
```diff
@@ -0,0 +1,822 @@
+{
```
<pre class="ql-syntax" spellcheck="false">Line #4.    
</pre><p>You forgot to compute the ra/dec from az/el here... but this actually needs to be done on the previous cell, when you are slewing to a new target while keeping the rotator at zero degrees.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

b1quint replied:
<p>Fixed this.</p>

tribeiro commented on procedures/LVV-T2572 - Rotator Duty Cycle Tests.ipynb:
```diff
@@ -0,0 +1,822 @@
+{
```
<p>This one is going to the same target 240 times, simply resetting the rotator angle. I think what we should probably do is move the az/el -&gt; ra/dec computation to inside the loop and maybe add a random az/el offset of about 1.5 degrees each. This would, in principle, perform a random walk around the initial az/el, which is pretty similar to what we will do on the main survey (in terms of telescope motion).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

tribeiro reviewed (approved):
These look pretty good! thank you for taking my initial comments into account. I left a couple more comments, please take a look before merging!

b1quint committed:
Apply minor fixes to the Rotator Duty Cycle Test notebook

These were changes requested from the PR review:
 - Fix cells that did not have the az/el->ra/dec conversion
 - Add an option to enable a random walk to the 2h survey

b1quint commented:
Applied changes as requested: 
- Fixed cells that did not contain alt/el -> ra/dec calculation
- Updated long-survey cell to add an option to run a random walk.
